Modules/Core/Common/include/itkMacro.h

```diff
@@ -1100,9 +1100,9 @@ compilers.
  * provides the GPU kernel source code as a const char*
  */
 #define itkGPUKernelClassMacro(kernel)                                                                                 \
-  /**\class kernel                                                                                                     \
-   * Workaround KWstyle bug                                                                                            \
-   * \ingroup ITKCommon                                                                                                \
+  /**\class kernel \                                                                                                                     \
```

Comment:
I don't think Doxygen will use that at all because it is configured to only used predefined macros: https://github.com/InsightSoftwareConsortium/ITK/blob/master/Utilities/Doxygen/doxygen.config.in#L2132

Reply:
Thanks @blowekamp ! It says "Workaround KWstyle bug", so is the KWstyle bug still relevant?

Reply:
I don't know. The usual way to work around a KWStyle issue is to add an override: https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Core/Common/ITKKWStyleOverwrite.txt

Reply:
Thanks for the information, Bradley. For now I would like to leave it this way. This pull request is just meant to fix the `itkDynamicCastInDebugMode` macro, and I think it's ready now! 😃